templates/includes/blog-embed.html

```diff
@@ -0,0 +1,26 @@
+<div class="blog-embed">
+    <img class="blog-embed__icon" src="/static/images/fontawesome-svgs/comment-white.svg" alt="">
+    <p class="blog-embed__icon-label" role="presentation" aria-hidden="true">
+        Blog
+    </p>
+
+    <h3 class="blog-embed__heading">
+        Paying for the privilege: a new Shakespeare discovery
+        <span class="sr-only">Blog</span>
```

Comment:
I wonder if "Blog" should appear before the heading text rather than before it, perhaps followed by a full stop or semi colon (in the same way that we present the type to sighted users before the heading). This would read as 

> Blog. Paying for the privilege: a new Shakespeare discovery

rather than 

> Paying for the privilege: a new Shakespear discovery Blog